### ISSUE TITLE
ASCII strings are not displayed properly in CPU window

### ISSUE THREAD

**Issue by Archer-Dante** [x64dbg/x64dbg#3340]
### Operating System

Windows 10 64bit, 19045.3208

### x64dbg Version

Apr 11 2024

### Describe the issue

ASCII symbols not displayed correctly in CPU window

how does the "hello" string looks like in Olly 1.10
![6bMq9GE](https://github.com/x64dbg/x64dbg/assets/7766366/ea9eb25b-8783-4506-9b5e-c76aed2ee09b)

how same place looks with x64dbg
![GqJ74ZL](https://github.com/x64dbg/x64dbg/assets/7766366/d0f4a572-155a-4004-9b59-472fc6c664be)

Also the bytes somehow grouped wrongly by 0000 instead of one 00 per line

### Steps to reproduce

Basically with any file, but it can be triggered with forcing any changes in string.
With olly after restart it fixes itself, but with x64dbg I didn't find any solution

**Comment by mrexodia** [comment]
This is an analysis that Olly does. You can achieve the same by selecting the bytes and going to Analysis -> Treat from selection as -> ASCII

**Comment by Archer-Dante** [comment]
Thanks, I've tried it, but it goes only 1 symbol per line\\byte, can I somehow display it as whole string like in old olly which classically ends with 00 bytes?
